ci-operator/config/openshift/route-monitor-operator/openshift-route-monitor-operator-master.yaml

```diff
@@ -1,35 +1,15 @@
-base_images:
-  base:
-    name: ubi
-    namespace: ocp
-    tag: "8"
-  openshift_release_golang-1.13:
-    name: release
-    namespace: openshift
-    tag: golang-1.13
-  openshift_release_golang-1.14:
-    name: release
-    namespace: openshift
-    tag: golang-1.14
 build_root:
-  image_stream_tag:
-    name: release
-    namespace: openshift
-    tag: golang-1.14
+  from_repository: true
 images:
 - dockerfile_path: Dockerfile
-  from: base
-  to: route-monitor-operator
+  to: unused
```

Comment:
why keep it then?

Reply:
Are you asking why we keep the image build when we’re not using the result? It’s to make sure it succeeds. I’m not sure about rmo (it’s a snowflake in several ways) but the other osd operators don’t have any other mechanism to validate the docker build at this stage.

Reply:
@petr-muller is we are deleting this, how would I test this works?

Reply:
yep makes sense